orga/app/components/routes/authenticated/campaign/list.hbs

```diff
@@ -15,12 +15,14 @@
               @field="name"
               @value={{@nameFilter}}
               @placeholder="Rechercher une campagne"
+              @aria-label="Rechercher une campagne"
               @triggerFiltering={{@triggerFiltering}}
             />
             <Table::HeaderFilterInput
               @field="creatorName"
               @value={{@creatorNameFilter}}
               @placeholder="Rechercher un créateur"
+              @aria-label="Rechercher un créateur"
```

Comment:
👍  là c'est surtout que le placeholder servait de `label`. Ce qui n'est pas une bonne pratique a11y,

Reply:
Oui je l'ai appris en lisant la règle a11y suivante : https://www.accede-web.com/en/guidelines/html-css-javascript/forms/use-the-aria-label-or-title-attribute-to-label-form-fields-that-dont-have-a-visible-label/